components/Snackbar/src/MDCSnackbarMessage.h

```diff
@@ -100,7 +100,7 @@ extern NSString * __nonnull const MDCSnackbarMessageBoldAttributeName;
 /**
   The color used for button text on the Snackbar in normal state.
 
-  Default is nil, but MDCRGBAColor(0xFF, 0xFF, 0xFF, 0.6f) will be set as the default color
+  Default is nil, but MDCRGBAColor(0xFF, 0xFF, 0xFF, (CGFloat)0.6) will be set as the default color
```

Comment:
tip: remove casting, make it just 0.6

Reply:
I wasn't sure why the comments used code that would compile, but I wanted to keep it as syntactically-correct as it was before.